keps/sig-apps/2879-ready-pods-job-status/README.md

```diff
@@ -157,14 +157,15 @@ pods that have the `Ready` condition.
 #### GA
 
 - Every bug report is fixed.
-- Explore setting different batch periods for regular pod updates versus
-  finished pod updates, so we can do less pod readiness updates without
-  compromising how fast we can declare a job finished.
-- The job controller ignores the feature gate.
+- Lock the feature-gate and document deprecation of the feature-gate
 
 #### Deprecation
 
-N/A
+In GA+1 release:
+- Job controller ignores the feature gate
```

Comment:
This can be done in the GA version.

Reply:
It's still useful to LockToDefault for anyone looking at the code.
Not sure if feature gates values are logged, but if they do, it would also be useful in that case.

Reply:
Aldo is right, at GA we block the feature gate to always on, in GA+2 we drop the feature from the code.

Reply:
So, IIUC, in GA we use `LockToDefault`, but keep the `if`s in code. GA+1 we do nothing, then GA+2 we delete both the declaration and the `if`s. @soltysh for confirmation.

Reply:
That's the practice, it ensures that if we need to revert GA graduation we have all in place.

Reply:
Ok, so one more small change - I have moved `Job controller ignores the feature gate` the GA+1 to GA+2. PTAL